airbyte-integrations/connectors/destination-mssql-v2/src/main/kotlin/io/airbyte/integrations/destination/mssql/v2/MSSQLQueryBuilder.kt

```diff
@@ -181,7 +180,7 @@ const val COUNT_FROM = """
     """
 
 class MSSQLQueryBuilder(
-    config: MSSQLConfiguration,
+    defaultSchema: String,
```

Comment:
why are we stringifying here?

Reply:
The `MSSQLQueryBuilder` only needs to the default schema and not the entire MSSQLConfiguration object thats we replaced it with the default schema